static/config/status-chain-genesis.json

```diff
@@ -3,16 +3,22 @@
      "AdAf150b905Cf5E6A778E553E15A139B6618BbB7": {
       "balance": "0x10F0CF064DD59200000"
     },
-    "65C01586aa0Ce152835c788aCe665e91Ab3527b8": {
+     "65C01586aa0Ce152835c788aCe665e91Ab3527b8": {
```

Comment:
Additional, unnecessary white space :)

Reply:
It is just align by first line "AdAf150b905Cf5E6A778E553E15A139B6618BbB7"...